frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/index.tsx

```diff
@@ -177,7 +207,7 @@ const AnalysisTable = memo(
           </Table>
         </TableContainer>
         <TablePagination
-          rowsPerPageOptions={[10, 25, 100]}
+          rowsPerPageOptions={maxRows !== undefined ? [] : [10, 25, 100]}
```

Comment:
Can you explain the behavior here? Seems a bit counter intuitive but maybe I missed something